rskj-core/src/test/java/co/rsk/core/BlockchainDummy.java

```diff
@@ -55,6 +56,11 @@ public Block getBestBlock() {
         return null;
     }
 
+    @Override
+    public void setBlockRecorder(BlockRecorder blockRecorder) {
+
```

Comment:
what if we throw `java.lang.UnsupportedOperationException` here?

Reply:
This is test code, but I'm fine either way.